app/feed/page.tsx

```diff
@@ -15,6 +19,24 @@ type Props = {
 };
 
 export default async function FeedPage({ searchParams }: Props) {
+  const filterEvetns = [
+    { title: "Repository", options: await fetchAllReposName() },
+    {
+      title: "Events",
+      options: [
+        "All",
+        "PullRequestReviewCommentEvent",
+        "PullRequestReviewEvent",
+        "MemberEvent",
+        "IssuesEvent",
+        "IssueCommentEvent",
+        "PullRequestEvent",
+        "PushEvent",
+        "ForkEvent",
+        "ReleaseEvent",
+      ],
+    },
+  ];
```

Comment:
For added type-safety

```suggestion
  ] as const;
```